src/components/dev-dot-content/mdx-components/mdx-tabs/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { ReactNode } from 'react'
+import Tabs, { Tab } from 'components/tabs'
+import s from './mdx-tabs.module.css'
+
+const MdxTab = Tab
```

Comment:
🎨 I think this can be simplified with an import alias. 

```suggestion
import Tabs, { Tab  as MdxTab } from 'components/tabs'
import s from './mdx-tabs.module.css'
```